symphony-bdk-core-invokers/symphony-bdk-core-invoker-api/src/main/java/com/symphony/bdk/core/api/invoker/JSON.java

```diff
@@ -0,0 +1,10 @@
+package com.symphony.bdk.core.api.invoker;
+
+import java.util.Map;
+
+public class JSON {
```

Comment:
Why did you created this class?

Reply:
The code generated by OpenAPI plugin uses it (and since we don't generate the support files I added it manually)

Reply:
I removed the unused method though (the import is still there in the generated code)